packages/grafana-ui/src/components/InfoBox/InfoBox.story.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { number } from '@storybook/addon-knobs';
+import { InfoBox } from './InfoBox';
+
+const InfoBoxStories = storiesOf('General/InfoBox', module);
+
+const getKnobs = () => {
+  const CONTAINER_GROUP = 'Container options';
+  // ---
+  const containerWidth = number(
+    'Container width',
+    800,
+    {
+      range: true,
+      min: 100,
+      max: 1500,
+      step: 100,
+    },
+    CONTAINER_GROUP
+  );
+
+  return { containerWidth };
+};
+
+InfoBoxStories.add('default', () => {
```

Comment:
just export a story like this;: https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/ThemeColors/ThemeColors.story.tsx#L13